internal/api/grpc/user/v2/user.go

```diff
@@ -182,86 +185,6 @@ func ifNotNilPtr[v, p any](value *v, conv func(v) p) *p {
 	return &pVal
 }
 
-func UpdateUserRequestToChangeHuman(req *user.UpdateHumanUserRequest) (*command.ChangeHuman, error) {
```

Comment:
This is refactoring.
I didn't want to make this already large file even larger by machine user implementations, so I moved some code to the new human.go file